.python_env_SAMPLE

```diff
@@ -14,18 +14,16 @@
 #
 # See https://github.com/kennethreitz/dj-database-url for URL formatting.
 #########################################################################
-# DATABASE_URL=sqlite:///db.sqlite3
+DATABASE_URL=postgres://cfpb:cfpb@postgres/cfgov
 
-# DATABASE_URL=postgres://cfpb:cfpb@postgres/cfgov
-# Required for Postgres dbshell in Django < 1.9.
-# PGPASSWORD=cfpb
-
-# The use of storage_engine here assumes use of MySQL 5.1. For MySQL 5.6
-# or later, use default_storage_engine instead.
-DATABASE_URL=mysql://v1:v1@mysql/v1?init_command='SET storage_engine=MYISAM'
-# Required for existing MySQL-specific database scripts.
-MYSQL_HOST=mysql
-MYSQL_ROOT_PW=root
-MYSQL_NAME=v1
-MYSQL_USER=v1
-MYSQL_PW=v1
+#########################################################################
+# Postgres env vars to make tools like `psql` easier to work with (and
+# in the case of PGPASSWORD, it also makes `manage.py dbshell` work without
+# a password in Django versions older than 1.9.)
```

Comment:
I think we need `PGPASSWORD` for `manage.py dbshell` to work inside the container because the db is on another "host" (container), causing it to prompt for a password. In stand-alone, it's localhost, and I'm guessing the default PG setup does not use password auth for localhost, so it's not an issue (but as you can see above in `.env_SAMPLE` I'm still adding those `PG*` env vars to make it easy to just run `psql` and friends with no args).